h/models/token.py

```diff
@@ -38,6 +39,15 @@ class Token(Base, mixins.Timestamps):
     #: A NULL value in this column indicates a token that does not expire.
     expires = sqlalchemy.Column(sqlalchemy.DateTime, nullable=True)
 
+    _authclient_id = sqlalchemy.Column('authclient_id',
+                                       postgresql.UUID(),
```

Comment:
Do you think there's any chance that we might want to use these authclient IDs in URLs one day / do you think we should use `URLSafeUUID` here? (I already missed this in the DB migration in the previous PR anyway...)